h2o-py/test-requirements.txt

```diff
@@ -12,13 +12,15 @@ twine==1.10.0
 urllib3==1.24.2
 grip==4.4.0
 wheel==0.30.0
-numpy==1.16.0
-scipy==1.0.0
-pandas==0.22.0
-statsmodels==0.8.0
+numpy==1.16.4
+scipy==1.2.2; python_version == '2.7'
+scipy==1.3.0; python_version > '2.7'
+pandas==0.24.2
+git+https://github.com/statsmodels/statsmodels#egg=975813e180c3b727e1d19ed40a54e0e343105dec # use patched version, latest release 0.9.0 is not compatible with other libs
 patsy==0.5.0
 sklearn
-scikit-learn==0.19.1
+scikit-learn==0.20.3; python_version == '2.7'
+scikit-learn==0.21.2; python_version > '2.7'
 six==1.11.0
```

Comment:
@michal-raska: these are a lot of library upgrades at once: are they all used by our tests? or demos maybe?
Even if all those upgrades look `minor` according to semver, some of them are major: for example, I'm aware of several backwards incompatibilities for scikit-learn 0.19 to 0.20.
That's why I'm wondering if those are necessary or just good-to-have.

Reply:
I was unable to get the lower versions installed for Python 3.7. This is the minimal set we need to upgrade in order to build the Python 3.7 images.